### PR TITLE
Add footnotes handling in docs

### PR DESCRIPTION
**Pull request by etiennebacher**
* Add footnotes handling (cf #182)
* Fix color theme for "back to top" button and scrollbars in code blocks

### CODE REVIEW

**Comment by etiennebacher** [comment]
actually, wait for @vincentarelbundock's refactor before merging this one too

File: docs/docs/stylesheets/extra.css [hunk 1]
```diff
@@ -84,7 +85,13 @@ Fonts are the same as pkgdown websites (otherwise tables not properly formatted)
 	scrollbar-color: var(--custom-scrollbar-col);
 }
 
+.md-top:hover {
+	background: var(--md-typeset-a-color-hover);
```
Comment (eitsupi) [inline docs/docs/stylesheets/extra.css:89]: It looks like tabs and spaces are mixed in this file.

**Review by eitsupi — APPROVED** [review]
Looks good!
Thanks.

File: docs/docs/stylesheets/extra.css (hunk 1 above)
Comment (etiennebacher) [reply docs/docs/stylesheets/extra.css:89]: It all appears the same to me in RStudio, do you know an automatic styler for CSS files?

Comment (eitsupi) [reply docs/docs/stylesheets/extra.css:89]: > It all appears the same to me in RStudio, do you know an automatic styler for CSS files?

Maybe [Prettier](https://prettier.io/)?

I believe the popular method is to specify the tab size with Editorconfig; I don't think RStudio IDE supported Editorconfig yet.

**Pull request merged**